packages/core/src/reducers/index.ts

```diff
@@ -3,13 +3,15 @@ import { validationReducer } from './validation';
 import { rendererReducer } from './renderers';
 import { fieldReducer } from './inputs';
 import { commonStateReducer, extractData, extractSchema, extractUiSchema } from './common';
+import { translationReducer } from './translation';
 
 export const appReducer: Reducer<any> = combineReducers(
   {
     'common': commonStateReducer,
     'validation': validationReducer,
     'renderers': rendererReducer,
-    'fields': fieldReducer
+    'fields': fieldReducer,
+    'translation': translationReducer
```

Comment:
I'd give an other name to the reducer, maybe i18n? We'll probably also store other i18n related bits there.